articles/azure-signalr/signalr-tutorial-build-blazor-server-chat-app.md

```diff
@@ -418,28 +419,31 @@ From Visual Studio 2019 version 16.2.0, Azure SignalR Service is build-in web ap
    }
    ```
 
-1. Configure Azure SignalR Service `ConnectionString` either in `appsetting.json` or with [Secret Manager](/aspnet/core/security/app-secrets?tabs=visual-studio#secret-manager) tool
+1. Configure the Azure SignalR Service connection string either in `appsettings.json` or with the [Secret Manager](/aspnet/core/security/app-secrets?tabs=visual-studio#secret-manager) tool.
 
 > [!NOTE]
-> Step 2 can be replaced by using [`HostingStartupAssembly`](/aspnet/core/fundamentals/host/platform-specific-configuration) to SignalR SDK.
-> 
-> 1. Add configuration to turn on Azure SignalR Service in `appsetting.json`
->    ```js
->    "Azure": {
->      "SignalR": {
->        "Enabled": true,
->        "ConnectionString": <your-connection-string>
->      }
->    }
->    ```
-> 
-> 1. Assign hosting startup assembly to use Azure SignalR SDK. Edit `launchSettings.json` and add a configuration like below inside `environmentVariables`.
->    ```js
->    "environmentVariables": {
->        ...,
->        "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.Azure.SignalR"
->      }
->    ```
+> Step 2 can be replaced with [Hosting Startup Assemblies](/aspnet/core/fundamentals/host/platform-specific-configuration) to SignalR SDK.
+
+1. Add the configuration to turn on Azure SignalR Service in `appsettings.json`
```

Comment:
This is not Step 3 but another option regarding Step 2. Please move back to Notes section or suggest other styles.